src/Redux/gameReducer.ts

```diff
@@ -1,11 +1,15 @@
 /* eslint-disable no-param-reassign */
 import { createReducer } from "@reduxjs/toolkit";
+
 import { IGameState } from "@/types";
-import { getProducts, getSearchGames, getThreeGames } from "./thunks/gameThunks";
+import { deleteCart, getCart, getProducts, getSearchGames, getThreeGames } from "./thunks/gameThunks";
 
 const initialState: IGameState = {
   items: [],
   searchGames: [],
+  cart: {
+    games: [],
+  },
 };
 
 export default createReducer(initialState, {
```

Comment:
You can do it in the following way:
```suggestion
export default createReducer(initialState, (builder) => {
  builder
    .addMatcher(
      isAnyOf(getThreeGames.fulfilled, getSearchGames.fulfilled),
      (state: IGameState, action: PayloadAction<IGame[]>) => {
        state.items = action.payload;
      }
    )
    .addMatcher(isAnyOf(getSearchGames.fulfilled), (state: IGameState, action: PayloadAction<IGame[]>) => {
      state.searchGames = action.payload;
    })
    .addMatcher(isAnyOf(getCart.fulfilled), (state: IGameState, action: PayloadAction<ICart>) => {
      state.cart = action.payload;
    });
});


```

**Notice**:  `isAnyOf` in the last 2 ones is redundant, you should find the way how to solve it correctly ;)